src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.Android.cs

```diff
@@ -259,6 +259,30 @@ public async Task SwappingOutAndroidContextDoesntCrash()
 			}, mauiContextStub2);
 		}
 
+		[Fact]
+		public async Task SearchHandlerRendersCorrectly()
```

Comment:
Does this test need to be moved? The code change is on iOS but this test will only run on Android

Reply:
Right, moved the test to run it in all the platforms.